src/Microsoft.OData.Edm/Microsoft.OData.Edm.txt

```diff
@@ -103,6 +103,7 @@ EdmModel_Validator_Semantic_InaccessibleType=The named type '{0}' could not be f
 EdmModel_Validator_Semantic_AmbiguousType=The named type '{0}' is ambiguous from the model being validated.
 EdmModel_Validator_Semantic_InvalidNavigationPropertyType=The type of the navigation property '{0}' is invalid. The navigation target type must be an entity type or a collection of entity type. The navigation target entity type must match the declaring type of the partner property.
 EdmModel_Validator_Semantic_NavigationPropertyWithRecursiveContainmentTargetMustBeOptional=The target multiplicity of the navigation property '{0}' is invalid. If a navigation property has 'ContainsTarget' set to true and declaring entity type of the property is the same or inherits from the target entity type, then the property represents a recursive containment and it must have an optional target represented by a collection or a nullable entity type.
+EdmModel_Validator_Semantic_RecursiveComplexTypedPropertyMustBeOptional=The type of the property '{0}' is invalid. If a complex typed property is of the same type (or base type) as its declaring type, then the property represents a recursive containment and the given property must be optional.

```

Comment:
The nullability of the property type is invalid. ....

Reply:
✅